gamemode/modules/drowning/cl_drowning.lua

```diff
@@ -0,0 +1,23 @@
+local drownTimeStart
+local shouldDrawDrowningElement = false
+
+net.Receive( "RVR_Player_Enter_Water", function()
+    local time = net.ReadFloat()
+
+    drownTimeStart = time
```

Comment:
is this  used anywhere, why does it exist

Reply:
It will be used for the hud element